app/main.py

```diff
@@ -2,0 +1,28 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
```

Comment:
It's better to use shlex.split function to split the command into tokens. It handles quotes and escape sequences.